### ISSUE TITLE
Make a more generic viewer

### ISSUE THREAD

**Issue by bnjbvr** [Phyks/medonay#6]
It seems the viewer currently needs a new file describing the view for each type of "EmailToken". How about doing a very similar viewer, that first goes through the entire list of entries for a given type, then identifies what the fields are, and accordingly creates columns for each field? (format would be inferred from the content, like "numbers only" => "number formatting", etc.)

**Comment by Phyks** [comment]
Indeed, it currently needs a definition of the way to render stuff. This lets us render different items in many different ways (tabular data vs calendar vs map etc).

Auto-render of unknown schemas looks not so easy to do in a reasonable way. In particular, it looks like a huge amount of work while there should not be many different schemas involved in a typical instance I guess (Reservation, Invoice, Events, …).

There might be some things from OpenDataSoft that we could borrow here, https://github.com/opendatasoft/ods-widgets for instance.
